Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -896,7 +896,7 @@
 
 #if !defined(ENABLE_TRACKER_DISPOSITION) \
     && (PLATFORM(IOS_FAMILY) \
-    || (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 120000))
+    || PLATFORM(MAC))
```

Comment:
Does this still mean anything, or can this be simplified further? I do not remember where macCatalyst goes.

Reply:
That seems correct, given this file is limited to Apple ports.